tests/data/custom/extra.cfg

```diff
@@ -0,0 +1,3 @@
+[custom_section]
+Different_case=val
+all_lower=something
```

Comment:
perhaps we should add another section related to my earlier comment.

Reply:
I can either update with a test to this patch monday, or add it on as an additional patch to give it more testing. I am making all the smaller changes today so you will be able to look monday

Reply:
Since the fix was part of this patch set, it'd make sense to me to have the test with it too to confirm the correct behaviour.